### DISCUSSION TITLE
Dockerfile alternative?

### DISCUSSION THREAD

**Discussion by Luxvao** [rancher-sandbox/rancher-desktop#2866] [Q&A] [answered]
I'm using this with containerd and nerdctl, but I can't find an alternative to Dockerfile?

**Comment by Luxvao** [comment]
Also I'm new to containerd and I'm used to Docker so I'm confused..

**Comment by jandubois** [comment, marked as answer]
`nerdctl` uses the same `Dockerfile` as the `docker` cli.

**Comment by Luxvao** [comment]
So it's named Dockerfile? Thats kinda weird ngl

**Comment by jandubois** [comment]
Please check the `nerdctl` documentation to understand what it does, and what level of compatibility with `docker` cli is currently provided: https://github.com/containerd/nerdctl

**Comment by Luxvao** [comment]
Okay. Thanks
